NanoVNASaver/skins_utils.py

```diff
@@ -0,0 +1,88 @@
+# Author Carl Tremblay - Cinosh07 AKA VA2SAJ
+#  Copyright (C) 2019.  Rune B. Broberg
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import logging
+import sys
+from os.path import join, dirname, abspath
+from .about import debug
+from .skins import Skins
+
+
+logger = logging.getLogger(__name__)
+
+def DARK_SKIN_MONOCHROME():
+    if debug == True:
+        return join(dirname(abspath(__file__)), 'skins/dark-monochrome.css')
+    else:
+        return sys._MEIPASS + '/skins/dark-monochrome.css'
+
+def DARK_SKIN_COLORED():
+    if debug == True:
+        return join(dirname(abspath(__file__)), 'skins/dark-colored.css')
+    else:
+        return sys._MEIPASS + '/skins/dark-colored.css'
+
+def LIGHT_SKIN_MONOCHROME():
+    if debug == True:
+        return join(dirname(abspath(__file__)), 'skins/light-monochrome.css')
+    else:
+        return sys._MEIPASS + '/skins/light-monochrome.css'
+
+def LIGHT_SKIN_COLORED():
+    if debug == True:
+        return join(dirname(abspath(__file__)), 'skins/light-colored.css')
+    else:
+        return sys._MEIPASS + '/skins/light-colored.css'
```

Comment:
The sys._MEIPASS variable is only used if packaged via pyinstaller. See the corresponding code in NanoVNASaver.py loading the icon for how to determine if the code is running within such an environment. "Normally" it should just load from a subdirectory. I think it's best to let that be a subdirectory of the application working directory? But I'm open to being educated on what's best practice.

Reply:
Done ready for the next commit